src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -181,12 +184,20 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id,
       // for each wrapper assembly, emit an assembly reference
       hr = metadata_builder.EmitAssemblyRef(
           method_replacement.wrapper_method.assembly);
-      RETURN_OK_IF_FAILED(hr);
+      if (FAILED(hr)) {
+        Warn("CorProfiler::ModuleLoadFinished: ", module_info.assembly.name,
+             ". Failed to emit wrapper assembly ref.");
+        return S_OK;
+      }
 
       // for each method replacement in each enabled integration,
       // emit a reference to the instrumentation wrapper methods
       hr = metadata_builder.StoreWrapperMethodRef(method_replacement);
-      RETURN_OK_IF_FAILED(hr);
+      if (FAILED(hr)) {
+        Warn("CorProfiler::ModuleLoadFinished: ", module_info.assembly.name,
+             ". Failed to emit wrapper method ref.");
```

Comment:
nit: Should be "Failed to STORE wrapper method ref" here

Reply:
If the `hr` has a failed status here, it's because we failed to emit the method ref. I'll change the message to say "Failed to emit and store...", since `StoreWrapperMethodRef()` does both.